packages/fxa-content-server/app/tests/spec/views/base.js

```diff
@@ -180,6 +180,13 @@ describe('views/base', function() {
   });
 
   describe('render', function() {
+    it('renders react component correctly', () => {
+      var renderReactComponent = sinon.spy();
+      view.render(renderReactComponent);
+      view.render().then(function() {
```

Comment:
can you add a `return` before this so that the promise is returned too, otherwise we don't know if the assertion fails.